### ISSUE TITLE
Setting the same process on two different chem plant slots will not work

### ISSUE THREAD

**Issue by basilicofresco** [Kerbalism/Kerbalism#642]
Hi! I noticed that when I set the same process on more than one slot of a chemical plant, then only the first one will actually work. The problem is visible on the planner and on the field. It affects all the converters (Chemical plant, Convert-O-Tron 125, Convert-O-Tron 250) and I guess all the processes. I just tried with water electrolysis and waste incinerator.
![kerbalism2](https://user-images.githubusercontent.com/31368421/83335233-97e9a300-a2ab-11ea-8457-e0905146c9ce.png)

Could the problem be related to the fact that there is just one switch on the pop-up window of the converter?
![kerbalism3](https://user-images.githubusercontent.com/31368421/83335720-f95f4100-a2ae-11ea-8349-86512fe8dabc.jpg)

I tested it with the usual clean installation with just KSP 1.9.1.2788 ("Science = false" in settings.cfg) + Breaking Ground 1.4.1 + Kerbalism 3.9. The problem is present both in career and science mode.
Thanks!

**Comment by gotmachine** [comment]
Yep, known issue. Won't fix as the configure module is going away in next major version.

**Comment by gotmachine** [comment]
To clarify a bit, I haven't investigated in detail, but due to how the configure module is implemented, I don't see how this can be fixed without a major refactor.

**Comment by Aebestach** [comment]
#948 has been fixed, and this issue will now be closed.
